scripts/debugger.test.js

```diff
@@ -0,0 +1,85 @@
+/**
+ * Copyright (c) 2017-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+/* @flow */
+
+// let child_process = require("child_process");
+import { UISession } from "../lib/debugger/mock-ui/UISession.js";
+import type { DebuggerCLIArguments } from "../lib/debugger/mock-ui/UISession.js";
+import fs from "fs";
+
+let expectedOutputsPrefix = ["Debugger is starting up Prepack...", "Prepack is ready"];
+let expectedOutputsSuffix = ["Prepack exited! Shutting down..."];
+
+function cleanDebuggerOutputs(outputs: Array<string>): Array<string> {
+  return outputs.filter(o => !o.includes("dbg")).map(o => o.replace(/\r?\n|\r/g, ""));
+}
+
+function cleanExpectedOutputs(expected: Array<string>): Array<string> {
+  return expectedOutputsPrefix.concat(expected).concat(expectedOutputsSuffix);
+}
+
+function generateArgs(prepackArguments: Array<string>, sourceFiles: Array<string>): DebuggerCLIArguments {
+  return {
+    adapterPath: "lib/debugger/adapter/DebugAdapter.js",
+    prepackRuntime: "bin/prepack.js",
+    sourceFiles: sourceFiles,
+    prepackArguments: prepackArguments,
+  };
+}
+
+function runTests(tests: Array<Array<string>>): void {
+  describe.each(tests)("Debugger Correctness Tests", (name, prepackArgs, inputSourceFiles, commands, expected) => {
+    test(name, done => {
```

Comment:
Prefer `it` to `test`, they're aliases, but everything else uses `it` iirc.

Reply:
Ah, does `it` also circumvent the `function test is not found` errors?